WalletWasabi.Daemon/Rpc/WasabiJsonRpcService.cs

```diff
@@ -293,10 +293,17 @@ public void SelectWallet(string walletName)
 	}
 
 	[JsonRpcMethod("stop", initializable: false)]
-	public async Task StopAsync()
+	public Task StopAsync()
 	{
 		// RPC terminating itself so it should not block this call while the RPC interface is stopping.
-		await Task.Run(() => TerminateService.Terminate());
+		Task.Run(async () =>
+		{
+			// Give some breathing room to send response to the user.
+			await Task.Delay(500).ConfigureAwait(false);
```

Comment:
Look, I don't like to have a delay because it is wrong, however it is even worse to have the exception so, if this solves the problem (or do it less likely) then I am okay this it.